python/lsst/meas/algorithms/measureApCorr.py

```diff
@@ -320,16 +272,25 @@ def plotApCorr(bbox, xx, yy, zzMeasure, field, title, doPause):
 
     Intended for debugging.
 
-    @param bbox  Bounding box (for bounds)
-    @param xx  x coordinates
-    @param yy  y coordinates
-    @param zzMeasure  Measured value of the aperture correction
-    @param field  Fit aperture correction field
-    @param title  Title for plot
-    @param doPause  Pause to inspect the residuals plot?  If False,
-                    there will be a 4 second delay to allow for
-                    inspection of the plot before closing it and
-                    moving on.
+    Parameters
+    ----------
+    bbox : `lsst.geom.Box2I`
+        Bounding box (for bounds)
+    xx : `array`, (N)
```

Comment:
xx and yy are `numpy.ndarray`. And you could bundle them with:

```
xx, yy : `numpy.ndarray`, (N)
    x and y coordinates
```

https://developer.lsst.io/python/numpydoc.html#shorthand